src/main/java/project/onlinebookstore/dto/cartitem/CreateCartItemRequestDto.java

```diff
@@ -0,0 +1,11 @@
+package project.onlinebookstore.dto.cartitem;
+
+import jakarta.validation.constraints.Positive;
+
+public record CreateCartItemRequestDto(
+        @Positive(message = "Field bookId can't be null")
```

Comment:
```suggestion
        @Positive(message = "Field bookId can't be null")
        @NotNull
```

"Field bookId can't be null" wrong message for that annotation